src/features/rewards/walletWrapper/index.tsx

```diff
@@ -411,10 +411,15 @@ export default class WalletWrapper extends React.PureComponent<Props, State> {
     const hasGrants = this.hasGrants(grants)
 
     let tokens = '0.0'
+    let date = ''
```

Comment:
nit: `date` could be moved to just before line 419 where it is used to make the code easier to read

Reply:
done